### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by JoshuaDelosSantos**
Hey Hey @Jackcox343 please check my variable naming and function SRP! 🙈

### CODE REVIEW

File: prac_06/used_cars.py [hunk 1]
```diff
@@ -9,10 +9,16 @@
 
 def main():
     """Demo test code to show how to use car class."""
-    my_car = Car(180)
+    my_car = Car("My Car", 180)
```
Comment (jackcox343) [inline prac_06/used_cars.py:12]: good naming

File: prac_06/used_cars.py [hunk 2]
```diff
@@ -9,10 +9,16 @@
...
     my_car.drive(30)
     print(f"Car has fuel: {my_car.fuel}")
     print(my_car)
+    print()  # Formatting purposes
```
Comment (jackcox343) [inline prac_06/used_cars.py]: nice use of commenting

File: prac_06/guitar_test.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+from prac_06.guitar import Guitar
+
+CURRENT_YEAR = 2023
+
+
+def main():
+    """Test code for Guitar class"""
+    my_guitar = Guitar("Gibson L-5 CES", 1922, 16035.40)
+    my_guitar_age = my_guitar.get_age(CURRENT_YEAR)
```
Comment (jackcox343) [inline prac_06/guitar_test.py]: good variable name

File: prac_06/guitar_test.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+"""
+
+from prac_06.guitar import Guitar
+
+CURRENT_YEAR = 2023
+
+
+def main():
+    """Test code for Guitar class"""
+    my_guitar = Guitar("Gibson L-5 CES", 1922, 16035.40)
```
Comment (jackcox343) [inline prac_06/guitar_test.py:13]: good variable name

**Comment by JoshuaDelosSantos** [comment]
Thanks @jackcox343 !

**Pull request merged**